controller.js

```diff
@@ -678,22 +668,33 @@ const getAllMenus = (req, res) => {
 
 const fetchAsset = async (req, res) => {
   let path = req.params[0];
-  s3.getObject(
-    {
-      Bucket: BUCKET_NAME,
-      Key: `assets/${path}`,
-    },
-    (err, data) => {
-      if (err) {
-        res.status(err.statusCode).send(err.message);
-      } else {
-        res.setHeader("Content-Type", "image/png");
-        res.send(data.Body);
-      }
-    }
-  );
+
+  getFile(`assets/${path}`)
+    .then((data) => {
+      res.setHeader("Content-Type", data.ContentType);
+      res.send(data.Body);
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(err.statusCode).send({
+        message: err.message
+      });
+    });
 };
 
+const uploadAsset = async (req, res) => {
```

Comment:
Can we make separate methods for uploading a restaurant image vs. a dish image, etc.?  

That way the frontend does't have to deal with resolving a path, the backend can handle that instead based on passed in restaurantID and dishID

Reply:
same for fetching assets.

Reply:
Let's do this in another pull request. I think there will be another load of changes coming in for this. Let's get this one merged first - I'm branching the new PR off of this one.